net/pfSense-pkg-haproxy-devel/Makefile

```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-haproxy-devel
-PORTVERSION=	0.59
-PORTREVISION=	20
+PORTVERSION=	0.60
+PORTREVISION=	1
```

Comment:
PORTREVISION line must be deleted when PORTVERSION changes

Reply:
Done